distribution/std-lib/Base/src/Time/Zone.enso

```diff
@@ -0,0 +1,59 @@
+from Base import all
+
+polyglot java import java.time.ZoneId
+polyglot java import java.time.ZoneOffset
+
+type Zone
+
+    ## A type representing a time zone.
+
+       A time zone can be eiter offset-based like "-06:00" or id-based like
+       "Europe/Paris".
+    type Zone internal_zone_id
+
+    ## Get the unique timezone ID.
+    zone_id : Text
+    zone_id = this.internal_zone_id . getId []
+
+## This method parses the ID producing a `Zone`.
+
+   > Example
+     Get Central European Time.
+         Zone.parse "CET"
+
+   > Example
+     Get Moscow time.
+         Zone.parse "Europe/Moscow"
+
+   > Example
+     Get time zone -06:00.
+         Zone.parse "-06:00"
+parse : Text -> Zone
+parse text = Zone (ZoneId.of [text])
+
+## The system default timezone.
+system : Zone
+system = Zone (ZoneId.systemDefault [])
+
+## Alias to `system`.
```

Comment:
Copy the doc from `system` – when the user uses the searcher, I want the to see the actual doc, not a pointer.